### DISCUSSION TITLE
BIG: Video missing in Lesson 7. The RSA algo. part 2. It skips calculating e and encrypting. There is a part missing.

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#321] [Course: Foundations of Private Computation]
Video missing in Lesson 7. The RSA algo. part 2. It skips calculating e and encrypting. There is a part missing.

**Comment by nigoshh** [comment]
Yes it seems that in "Concept 3: The RSA Algorithm, Part 2" there is the wrong video, "Cryptography 4: The RSA Algorithm, Part 3". Same in Concept 4 (video of Part 4 instead of Part 3).
